CONTRIBUTING.md

```diff
@@ -15,8 +15,9 @@ If you are new to contributing to open source, please read the Open Source Guide
     - [Issues](#issues)
     - [Pull Requests](#pull-requests)
     - [Branching Strategy](#branching-strategy)
-    - [Conflict Resolution](#conflict-resolution)
+    - [Branching Strategy](#branching-strategy-1)
```

Comment:
This file is out of scope.

Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit